src/ui/public/state_management/state_monitor.js

```diff
@@ -50,6 +55,17 @@ function stateMonitor(state, defaultState) {
   };
 
   return {
+    setDefault(defaultState) {
+      // update the originalState and apply ignoredProps
+      if (defaultState) {
+        setOriginalState(defaultState);
+        filterState(originalState);
```

Comment:
I just realized we never make use of the returned `state` in `filterState`. Can we remove that `return`? And can we also rename it `removeIgnoredProps` or `ignorePropsInState` to better reflect what it's doing?